python_modules/dagster/dagster/_grpc/client.py

```diff
@@ -128,17 +128,34 @@ def _get_response(
             stub = DagsterApiStub(channel)
             return getattr(stub, method)(request, metadata=self._metadata, timeout=timeout)
 
+    def _raise_grpc_exception(self, e: Exception, timeout, custom_timeout_message=None):
+        if isinstance(e, grpc.RpcError):
+            if e.code() == grpc.StatusCode.DEADLINE_EXCEEDED:
+                raise DagsterUserCodeUnreachableError(
+                    custom_timeout_message
+                    or f"User code query timed out due to taking longer than {timeout} seconds to complete."
```

Comment:
nit: user code -> code location server

```suggestion
                    or f"The query to the code location server timed out due to taking longer than {timeout} seconds to complete."
```